src/backends/sqlite.rs

```diff
@@ -9,11 +8,13 @@ use rmp_serde::{Serializer, Deserializer};
 use {Result, KVStore, Record};
 use tx::TxRaw;
 
-#[derive(Clone)]
 pub struct SqliteStore {
-    conn: Arc<sql::Connection>,
+    conn: sql::Connection,
 }
 
+// FIXME: this is irresponsible!
+unsafe impl ::std::marker::Sync for SqliteStore {}
```

Comment:
Please do. Would be nice to get rid of this. For some more information, see [this issue](https://github.com/jgallagher/rusqlite/issues/188).